articles/search/search-howto-complex-data-types.md

```diff
@@ -244,6 +244,71 @@ To filter on a complex collection field, you can use a **lambda expression** wit
 
 As with top-level simple fields, simple subfields of complex fields can only be included in filters if they have the **filterable** attribute set to `true` in the index definition. For more information, see the [Create Index API reference](/rest/api/searchservice/create-index).
 
+Azure search has the limitation wherein the complex object in all the collections total in a document cannot exceeed 3000
```

Comment:
make sure it says that the customer needs to validate whether this workaround meets their scenario requirements.

Reply:
Added it as suggested "Storing these complex values as strings avoids the limitation. The customer needs to validate whether this workaround meets their scenario requirements."